Mathlib/RingTheory/Localization/FractionRing.lean

```diff
@@ -222,12 +222,15 @@ such that `z = f x * (f y)⁻¹`. -/
 noncomputable def lift (hg : Injective g) : K →+* L :=
   IsLocalization.lift fun y : nonZeroDivisors A => isUnit_map_of_injective hg y
 
-theorem lift_unique (hg : Function.Injective g) {f : K →+* L}
+theorem lift_unique
+    (hg : Function.Injective g)
+    {f : K →+* L}
     (hf1 : ∀ x, f (algebraMap A K x) = g x) : IsFractionRing.lift hg = f :=
   IsLocalization.lift_unique _ hf1
 
 /-- Another version of unique to give two lift maps should be equal -/
-theorem lift_unique' {f1 f2 : K →+* L}
+theorem lift_unique'
+    {f1 f2 : K →+* L}
```

Comment:
I don't think putting these variables on their own lines is the preferred style. For instance, see how some of the theorems on this page are structured: https://leanprover-community.github.io/contribute/style.html

Reply:
emm I think you're right. Sorry.